### PR TITLE
feat(chore): update nodes to new runners for krt v2

### PR DESCRIPTION
**Pull request by alexrodfe**
## Why
Now that the krt v2 support in both kre and runners is ready, we need to update this repo's nodes to support as well krt v2 runners.

## What
- Update runner images in Dockerfile
- Update handler methods in all nodes

### CODE REVIEW

**Review by aacecandev — APPROVED** [review]

File: runtime/src/etl/main.py [hunk 1]
```diff
@@ -5,24 +5,24 @@
 
 
 def init(ctx):
-    ctx.logger.info('Initializing runner')
```
Comment (aacecandev) [inline runtime/src/etl/main.py:8]: This kind of problems could be easily solved using pre-commit

File: runtime/src/etl/main.py [hunk 2]
```diff
@@ -5,24 +5,24 @@
...
     req = Request()
     data.Unpack(req)
-    encoder: dict = ctx.get('encoder')
-    return new_etl_output(encoder, req)
+    encoder: dict = ctx.get("encoder")
+    return new_etl_output(ctx, encoder, req)
 
 
-def new_etl_output(encoder, req: Request) -> EtlOutput:
+def new_etl_output(ctx, encoder, req: Request):
```
Comment (aacecandev) [inline runtime/src/etl/main.py:25]: I think we should keep using typing and explicitly telling what kind of type functions return

Comment (alexrodfe) [reply runtime/src/etl/main.py:25]: That's the thing, they no longer return any type

File: runtime/src/etl/main.py (hunk 1 above)
Comment (alexrodfe) [reply runtime/src/etl/main.py:8]: Oh, I thought this was my automatic code formatter.

**Review by dfinteligenz — APPROVED** [review]

File: runtime/src/etl/main.py [hunk 3]
```diff
@@ -39,4 +42,5 @@ def new_etl_output(encoder, req: Request) -> EtlOutput:
     model_input.room_type = encoder["room_type"][req.room_type]
     model_input.neighbourhood = encoder["neighbourhood"][req.neighbourhood]
 
-    return res
+    await ctx.send_output(res)
+    return
```
Comment (blopezpi) [inline runtime/src/etl/main.py:46]: It's not needed to use return, in python a function without return always return None.

File: runtime/src/output/main.py [hunk 4]
```diff
@@ -2,46 +2,50 @@
...
     res = Response()
     res.success = True
-    res.message = 'Predicted market price'
+    res.message = "Predicted market price"
     res.market_price = market_price
     res.category = label
 
-    return res
+    await ctx.send_output(res)
+    return
```
Comment (blopezpi) [inline runtime/src/output/main.py:45]: Same here.

File: runtime/src/save-prediction-metric/main.py [hunk 5]
```diff
@@ -13,6 +13,7 @@ async def handler(ctx, data):
 
     res = SaveMetricResponse()
     res.success = True
-    res.message = 'Prediction metric saved'
+    res.message = "Prediction metric saved"
 
-    return res
+    await ctx.send_output(res)
+    return
```
Comment (blopezpi) [inline runtime/src/save-prediction-metric/main.py:19]: Same here.

**Review by blopezpi — APPROVED** [review]

File: runtime/src/etl/main.py [hunk 6]
```diff
@@ -1,28 +1,31 @@
...
     data.Unpack(req)
-    encoder: dict = ctx.get('encoder')
-    return new_etl_output(encoder, req)
+    encoder: Dict = ctx.get("encoder")
+    await new_etl_output(ctx, encoder, req)
+    return
 
 
-def new_etl_output(encoder, req: Request) -> EtlOutput:
+async def new_etl_output(ctx, encoder: Dict[str, Any], req: Request):
```
Comment (aacecandev) [inline runtime/src/etl/main.py:28]: We should explicitly tell that the function return None `async def ... () -> None:`

**Pull request merged**